### ISSUE TITLE
Content margins in sidebar

### ISSUE THREAD

**Issue by den-is** [jrolfs/sodarized#10]
Hi there, 
I really like minimalist view of sidebar without folder icons in Sublime Text 3.
But this didn't help me: `"soda_folder_icons": false`

So i have disabled icons via editing: Sodarized Light 3.sublime-theme
`// Sidebar folder closed
    {
        ...
        "content_margin": [0, 0]
    },`

But content of folders looks too far in folder....left margin is huge.
I can't find any settings which can fix it in theme file or wherever else.
Same happens on both linux and win machines
If there any chance to fix it ? 
![Sublime sidebar](http://i68.tinypic.com/20785ee.jpg)

**Comment by csenn** [comment]
Yeah, I love the theme, but I agree the sidebar folder margin is awkward.

**Comment by dnlglsn** [comment]
Taken from mvhenderson's answer here: https://github.com/jrolfs/sodarized/issues/5
----
Create file `.../Sublime Text 3/Packages/User/Sodarized Dark 3/Sodarized Dark 3.sublime-theme` with the following content:
```python
[
   {
       "class": "icon_file_type",
       "content_margin": [0,0]
   },
   {
       "class": "icon_folder",
       "content_margin": [0,0]
   },
   {
       "class": "icon_folder_loading",
       "content_margin": [0,0]
   }
]
```
